<!-- Launch file to store the names of dVRK topics -->
<launch>
	<group ns="dvrk_topics">



        <param name="current_state" type="string" value="status" />

        <param name="set_position_joint" type="string"
                                    value="servo_jp" />

        <param name="state_joint_current" type="string"
                                    value="measured_js" />

	<param name="set_position_cartesian" type="string"
                                    value="servo_cp" />

	<param name="position_cartesian_current" type="string"
                                    value="measured_cp" />

        <param name="set_position_jaw" type="string"
                                    value="jaw/servo_jp" />

        <param name="state_jaw_current" type="string"
                                    value="jaw/measured_js" />

	<param name="error" type="string" value="error" />

	<param name="warning" type="string" value="warning" />


	
	</group>
	
</launch>
